Formula/r/rqlite.rb

```diff
@@ -18,8 +19,15 @@ class Rqlite < Formula
   depends_on "go" => :build
 
   def install
+    ldflags = %W[
+      -s -w
+      -X github.com/rqlite/rqlite/v8/cmd.Commit=#{Utils.git_head}
```

Comment:
We may wish to parameterize v8, not sure. What is the right way to handle the eventual release of rqlite v9? I expect a new major release during 2025.

Reply:
We can use the `version.major` to account for this.